rudder-web/src/main/resources/configuration.properties.sample

```diff
@@ -68,6 +68,10 @@ rudder.dir.lock=/var/rudder/lock/
 #
 rudder.endpoint.cmdb=http://localhost:8080/endpoint/upload/
 
+#
+# Location of the relay api used by rudder webapp
+#
+rudder.server.relay.api=localhost
```

Comment:
Here, we need to parametrize the whole address - nothing tell us what would be configured in multi-server in the future. 
I don't see any simple way to have: ```https://localhost/rudder/relay-api/remote-run/nodes/${nodeId}```, but we can use: ```https://localhost/rudder/relay-api/remote-run/nodes/%s" with a comment stating that "%s will be replaced by the node id".